CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * Enable effective call number column sorting in Open Loans screen. Refs UIU-3002.
 * User Information in User Edit to display profile picture and update button set. Refs UIU-3005.
 * Update request header for pay several Fees/fines. Refs UIU-3040.
+* Changing user type confirmation modal. Refs UIU-2969.
```

Comment:
Please update the description to state that it is for ECS enabled env.

Reply:
done